File: fizzbuzz.erl
% http://imranontech.com/2007/01/24/using-fizzbuzz-to-find-developers-who-grok-coding/

-module(fizzbuzz).

-compile([export_all]).

fizzbuzz() ->
	[ case {X rem 3, X rem 5} of {0, 0} -> fizzbuzz; {0, _} -> fizz; {_, 0} -> buzz; _ -> X end || X <- lists:seq(1,100)].

File: fizzbuzz.py
def FizzBuzz(i):
	Remainders = (i % 3, i % 5)
	if Remainders == (0,0):
		return "fizzbuzz"
	elif Remainders[0] == 0:
		return "fizz"
	elif Remainders[1] == 0:
		return "buzz"
	return i

print [FizzBuzz(i) for i in range(1,101)]

